src/Log/LogService.php

```diff
@@ -369,7 +369,8 @@ public function getLog($name = null)
      */
     protected function createLogger($format, $priority, $file)
     {
-        $formatter = new \Zend_Log_Formatter_Simple($format);
+        $formatWithRunId = $this->prepareFormat($format);
```

Comment:
Please change to `$preparedFormat`. "WithRunId" only applies currently, but might not in the future.